<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentadorOlivosDialogBase</class>
 <widget class="QDialog" name="SegmentadorOlivosDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>293</height>
   </rect>
  </property>
  <widget class="QLabel" name="labelEstado">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Capa TIFF.</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboRaster">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAOI">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capa AOI:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboAOI">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsuFactor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Factor Otsu:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinOtsuFactor">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>96</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.50</double>
   </property>
   <property name="maximum">
    <double>2.00</double>
   </property>
   <property name="singleStep">
    <double>0.05</double>
   </property>
   <property name="value">
    <double>1.00</double>
   </property>
  </widget>


  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>250</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTerminarMuestras">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminar Selección</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIniciarMuestras">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Selección de Parcela</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
